src/pages/patientView/copyNumberAlterations/column/AnnotationColumnFormatter.tsx

```diff
@@ -56,19 +58,32 @@ export default class AnnotationColumnFormatter
     * Returns an ICivicEntry if the civicGenes and civicVariants have information about the gene and the mutation (variant) specified. Otherwise it returns
     * an empty object.
     */
-    public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, civicVariants:ICivicVariant): ICivicEntry | null
+    public static getCivicEntry(copyNumberData:DiscreteCopyNumberData[], civicGenes:ICivicGene, 
+                                civicVariants:ICivicVariant): ICivicEntry | null
     {
+        let civicEntry = null;
         let geneSymbol: string = copyNumberData[0].gene.hugoGeneSymbol;
         let geneVariants: {[name: string]: ICivicVariantData} = civicVariants[geneSymbol];
         let geneEntry: ICivicGeneData = civicGenes[geneSymbol];
-        let civicEntry = null;
         //Only return data for genes with variants or it has a description provided by the Civic API
         if (geneVariants || (geneEntry && geneEntry.description !== "")) {
             civicEntry = buildCivicEntry(geneEntry, geneVariants);
         }
 
         return civicEntry;
     }
+    
+    public static getCivicStatus(civicGenesStatus:"pending" | "error" | "complete", civicVariantsStatus:"pending" | "error" | "complete"): "pending" | "error" | "complete"
+    {
+    if (civicGenesStatus == "error" || civicVariantsStatus == "error") {
```

Comment:
please always use === instead of double equals.   just a good habit to get into.  (we need to enforce this with linting.)